bcipy/language/main.py

```diff
@@ -18,26 +18,28 @@ def __str__(self):
         return self.value
 
 
-class LanguageModel(ABC):
-    """Parent class for Language Models."""
+class BciPyLanguageModel(Protocol):
```

Comment:
This should just be `LanguageModel`. It's already name-spaced in the bcipy package. As far as I can tell we never import the textpredict `LanguageModel` base class, but if we did we can use an import alias (https://docs.python.org/3/reference/simple_stmts.html#import) to disambiguate.